src/navigations/root-stack.js

```diff
@@ -332,7 +355,7 @@ const AuthenticatedNavigator = () => {
         />
         <AuthenticatedStack.Screen
           name="SignedChatScreen"
-          component={withSafeAreaView(withKeyboardWrapper(SignedChatScreen))}
+          component={withKeyboardWrapper(SignedChatScreen)}
           options={{headerShown: false}}
         />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `withSafeAreaView` wrapper has been removed from the `SignedChatScreen` component. Ensure that this does not affect the layout of the `SignedChatScreen` on devices with notches or rounded corners (like iPhone X and above).


<!-- This is an auto-generated comment by OSS CodeRabbit -->